apps/aocbot/deploy.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: aocbot
+  namespace: default
+  labels:
+    app: aocbot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: aocbot
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  template:
+    metadata:
+      name: aocbot
+      labels:
+        app: aocbot
+    spec:
+      containers:
+        - name: aocbot
+          image: ghcr.io/wisvch/aocbot:20241205-287e5bf # {"$imagepolicy": "flux-system:aoch"}
```

Comment:
Perhaps change the image policy name, unless you want the other leaderboard

Reply:
Good one on it

Reply:
Done